connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -82,7 +82,7 @@
     void connectorConfig(String connName, Callback<Map<String, String>> callback);
 
     /**
-     * Set the configuration for a connector. This supports creation, update, and deletion.
+     * Set the configuration for a connector. This supports creation and updating.
```

Comment:
Or perhaps we should raise an exception if the config is null? Are you worried about people who have coded against the `Herder` directly?